src/DynamoCore/Graph/Nodes/NodeModel.cs

```diff
@@ -1738,12 +1738,17 @@ public virtual void ClearErrorsAndWarnings()
         /// </summary>
         public virtual void ClearInfoMessages()
         {
+            // It is very unlikely a node could be in both info state or persistent info state from the current design
```

Comment:
do you mean "unlikely a node could be in both info state *and* persistent info state" - since we're talking about logic here I want to be clear.

Reply:
yes